lib/cli/index.js

```diff
@@ -1,37 +1,85 @@
 const { resolve } = require('path');
 const { existsSync } = require('fs');
 const { printHelp, printVersion } = require('./print');
+const { getFiles } = require('./get-files');
 
-// Removes `node` and `__filename` from arguments list
-const nonTrivialArguments = process.argv.slice(2);
+const CLI_OPTION = {
+  HELP: 'help',
+  VERSION: 'version',
+  EXTENSIONS: 'extensions',
+  DIRECTORY: 'directory',
+};
 
-const showHelp = nonTrivialArguments.find(item => item === '-h' || item === '--help');
-const showVersion = nonTrivialArguments.find(item => item === '-v' || item === '--version');
+function getFlagValue(argv, flag, shortcut) {
+  return shortcut
+    ? !!argv.find(item => item === flag || item === shortcut)
+    : !!argv.find(item => item === flag);
+}
+
+function getArgumentValue(argv, option, shortcut) {
+  const optionArgvItemIndex = argv.findIndex(item => item.includes(option) || item.includes(shortcut));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 103. Maximum allowed is 100.](https://app.codacy.com/manual/spagnolo.manu/titef/pullRequest?prid=5052259)